ateam_common/CMakeLists.txt

```diff
@@ -22,14 +23,13 @@ add_library(${PROJECT_NAME} SHARED
   src/multicast_receiver.cpp
   src/parameters.cpp
   src/status.cpp
-  src/team_info_listener.cpp
-  src/game_state_listener.cpp
+  src/game_controller_listener.cpp
 )
-set_target_properties(${PROJECT_NAME} PROPERTIES CXX_STANDARD 17)
+set_target_properties(${PROJECT_NAME} PROPERTIES CXX_STANDARD 20)
 ament_target_dependencies(${PROJECT_NAME}
   rclcpp
   rclcpp_components
-  Eigen3
```

Comment:
Eigen is still used in this package. Can anyone remember why we removed it here?

Reply:
I added this back in... on line 13, do we need the `eigen3_cmake_module`? When looking at the package here on GitHub https://github.com/ros2/eigen3_cmake_module, it looks like this is to address issues with Ubuntu Bionic (18.04 LTS) which I assume none of us are using with ROS2 Humble, since it's not compatible.

Reply:
Does a clean build work without it?
I know the readme mentions Bionic, but I don't think the Eigen setup has changed in recent releases, so it might still be relevant.

Reply:
A clean build with line 13 commented out passes our tests and successfully launches the simulation bringup file. But maybe not worth removing, just to be safe.

Reply:
Fair enough. Good to know though.